lib/api/core/validation/types/geoPoint.js

```diff
@@ -0,0 +1,40 @@
+var
+  BaseConstructor = require('../baseType'),
+  geoUtil = require('../../../dsl/geoutil');
```

Comment:
:warning: This won't work with the new DSL.
We should see eachother to settle a way to modify both the current DSL and the future one to properly expose these utility methods.
If different modules use them, then we may have to store them elsewhere in the core architecture.